railties/RDOC_MAIN.rdoc

```diff
@@ -1,48 +1,55 @@
-== Welcome to \Rails
+= Welcome to \Rails
 
-\Rails is a web-application framework that includes everything needed to
+== What's Rails
```

Comment:
Let's use `\Rails`. And please confirm generated pages after `bundle exec rake rdoc && open doc/rdoc/index.html`.